### DISCUSSION TITLE
不同网盘之间使用rclone 复制时多一层目录

### DISCUSSION THREAD

**Discussion by 9k001** [AlistGo/alist#2170] [Invalid question]
version:  v3.2.1
问题: 
    使用alist挂载网盘,不同网盘之间使用 rclone 进行同步,对应文件在同步时，会多创建一层 以文件名为名的 文件夹。不知道这个问题怎么解决。
    即 
    a挂载 (源数据) 
    目录下有 aa.txt  ab.txt  ac.txt  。
    b挂载 (备份)
    rclone 设置 webdav config 文件
     使用 
      rclone copy  webdav:a webdav:b 或 rclone sync webdav:a webdav:b 进行同步时 
     会报错 ，但是查看 b挂载中 复制已经完成，但是多了一层 以 文件名 命名的文件夹。
     同步后 文件完整路径如下    b/aa/aa.txt  b/ab/ab.txt  b/ac/ac.txt
     求问这个问题怎么解决
![image](https://user-images.githubusercontent.com/63529476/199159066-9a83b52a-040f-4490-88d3-17bcc16033c8.png)
![image](https://user-images.githubusercontent.com/63529476/199161614-51f54f4a-dab3-4c12-934c-63016bdd8a66.png)

**Comment by githubgj** [comment]
当我从家里的NAS利用rclone备份数据到百度网盘，也出现了和你一样的情况。请问，后来这个问题解决了没？谢谢

**Reply by 9k001** [reply]
最后 阿里云盘 使用  Aliyundrive-Webdav 挂载出来 解决了这个问题

**Comment by furacas** [comment]
有用alist解决的思路吗。
